src/LazySet.jl

```diff
@@ -341,6 +345,61 @@ function ==(X::LazySet, Y::LazySet)
 
     return true
 end
+"""
+    ≈(X::LazySet, Y::LazySet)
+
+Return whether two LazySets of the same type are approximately equal.
+
+### Input
+
+- `X` -- any `LazySet`
+- `Y` -- another `LazySet` of the same type as `X`
+
+### Output
+
+- `true` iff `X` is equal to `Y`.
+
+### Notes
+
+The check is purely syntactic and the sets need to have the same base type.
+For instance, `X::VPolytope ≈ Y::HPolytope` returns `false` even if `X` and `Y`
+represent the same polytope.
+However `X::HPolytope{Int64} ≈ Y::HPolytope{Float64}` is a valid comparison.
+
+### Algorithm
+
+We recursively compare the fields of `X` and `Y` until a mismatch is found.
+
+### Examples
+
+```jldoctest
+julia> HalfSpace([1], 1) ≈ HalfSpace([1], 1)
+true
+
+julia> HalfSpace([1], 1) ≈ HalfSpace([1.00000001], 0.99999999)
+true
+
+julia> HalfSpace([1], 1) ≈ HalfSpace([1.0], 1.0)
+true
+
+julia> Ball1([0.], 1.) ≈ Ball2([0.], 1.)
+false
+```
+"""
+function ≈(X::LazySet, Y::LazySet)
+    # if the common supertype of X and Y is abstract, they cannot be compared
+    if isabstracttype(promote_type(typeof(X), typeof(Y)))
+        return false
+    end
+
+    for f in fieldnames(typeof(X))
+        if !(getfield(X, f) ≈ getfield(Y, f))
```

Comment:
```suggestion
        if !_isapprox(getfield(X, f), getfield(Y, f))
```
so that it uses the defaults defined in `comparisons.jl` (we will add a feature so they can be tuned, eventually..)

Reply:
This does not work because `_isapprox` is only defined between numbers.